ui/desktop/electron-app/config/desktop.js

```diff
@@ -2,6 +2,17 @@ const fs = require('fs');
 const path = require('path');
 const { isLinux } = require('../src/helpers/platform.js');
 
+const returnSemVerFromReleaseVersion = (releaseVersion) => {
+  let [major, minor, patch] = releaseVersion.split('.').slice(0, 3);
+
+  // Delete, if present, v from Major
+  if (!parseInt(major)) {
+    major = major.replace('v', '');
+  }
+
+  return major.concat('.', minor, '.', parseInt(patch));
```

Comment:
Do we `parseInt(patch)` to basically remove any text that may have been attached to it? e.g. `1-beta` in `3.2.1-beta`? If so, perhaps we could add a comment to explain that?

Could this also have just used a template string?
```suggestion
  return `${major}.${minor}.${parseInt(patch)}`
```

Reply:
Yes.

- The `parseInt` for `patch` was add for that exact reason. Good catch, added a comment.
- I also move the comment to explain `returnSemVerFromReleaseVersion`.